runtime/src/main/java/io/quarkiverse/cxf/CXFClientInfos.java

```diff
@@ -28,7 +30,7 @@ public static CXFClientInfo fromSei(String sei) {
 
     public CXFClientInfo getClientInfoBySei(String sei) {
         for (CXFClientInfo clientInfo : cxfClientInfos) {
-            if (clientInfo.getSei() == sei) {
+            if (Objects.equals(clientInfo.getSei(), sei)) {
```

Comment:
The original code yielded true when both strings were null, but I'm not sure if that was actually a requirement. This code maintains the same behaviour but let me know if I should change it to a simple `String.equals()`.